src/Field.tsx

```diff
@@ -21,6 +21,8 @@ export function Field<T extends FieldProps>({
   theme: _theme,
   disabled,
   className,
+  renderIcon,
+  renderIconBefore,
```

Comment:
Maybe this should be more generic. For example: `renderBeforeInput` or `renderAfterInput`? I don't feel too strongly about this.

However, I guess I'm imagining a scenario where someone wanted to put a `$` before an input or something.

Reply:
I'd recommend using a $ icon, in that case, and rely on the label to make sure its adequately described as a dollar amount.

I did consider something more generic, but if someone puts any-old-content in there, theres no way to make the appropriate spacing for it without knowing the exact size. At least with this, we generally know an Icon will be squarish and the extra margins I'll add on the CSS side should accommodate.

This corresponding PR in StackupUI handles the spacing https://github.com/promptworks/stackup-ui/pull/75

Reply:
Ugh, bootstrap. Do we have to??

Reply:
Okay, we can skip it for now, though it does seems like a small sacrifice. We can add a config later to change the order of things.